crates/fda/Cargo.toml

```diff
@@ -11,25 +11,26 @@ include = ["openapi.json", "/src", "build.rs", "COPYRIGHT", "README.md"]
 
 [dependencies]
 log = "0.4"
-clap = { version = "4.5.17", features = ["derive", "env", "color"] }
-clap_complete = { version = "4.5.26", features = ["unstable-dynamic"] }
-progenitor-client = { version = "0.7.0" }
-reqwest = { version = "0.11", features = ["json", "stream"] }
+clap = { version = "4.5", features = ["derive", "env", "color"] }
```

Comment:
isn't it a better approach to specify the version?

Reply:
I think we have it in cargo.lock